### ISSUE TITLE
Fix Cypress errors

### ISSUE THREAD

**Issue by racedale** [Rebuild-Black-Business/RBB-Website#88]
As a developer 
I want the tests to be passing to be able to ensure quality to our users

There's a few errors around navigation and a11y issues

Note:
`npm run test:e2e:ci` will execute Cypress tests

**Comment by marcuslyons** [comment]
I can take this on.

Looks like 3 failing tests in Navigation:
![image](https://user-images.githubusercontent.com/10541922/84175733-9f444580-aa3d-11ea-81ca-5b727ef8ace1.png)

And 6 a11y violations detected:
![image](https://user-images.githubusercontent.com/10541922/84175798-b3884280-aa3d-11ea-9d98-f579a7748299.png)
